### PR TITLE
fix(docs): some example docs fixes

### PR DESCRIPTION
**Pull request by devcode8**
https://github.com/fetchai/docs-v2/issues/645

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,16 +1,12 @@
 lockfileVersion: '6.0'
 
-settings:
```
Comment (devjsc) [inline pnpm-lock.yaml:3]: How come lock has changed?

Comment (devcode8) [reply pnpm-lock.yaml:3]: don't know , I only pushed the example docs files..

Comment (gautamgambhir97) [reply pnpm-lock.yaml:3]: @devcode8 you accidently push the lock file as well

Comment (devcode8) [reply pnpm-lock.yaml:3]: oohk got it , I think it didn't ignore by my .gitignore file

**Review by FelixNicolaeBucsa — APPROVED** [review]

**Pull request merged**